### ISSUE TITLE
Puro crash on create and errors on others

### ISSUE THREAD

**Issue by damywise** [PixelToast/puro#5]
## `puro create <name> <version>`
```zsh
[E] Exception while Pre-caching engine
[E] Assertion failed: "Unexpected result from sysctl: `1\n`"
    #0      EngineBuildTarget.query (package:puro/src/env/engine.dart:83)
    <asynchronous suspension>
    #1      downloadSharedEngine (package:puro/src/env/engine.dart:182)
    <asynchronous suspension>
    #2      createEnvironment.<anonymous closure>.<anonymous closure> (package:puro/src/env/create.dart:127)
    <asynchronous suspension>
    #3      runOptional (package:puro/src/logger.dart:139)
    <asynchronous suspension>
    
[x] Assertion failed: "Unexpected result from sysctl: `1\n`"
    #0      EngineBuildTarget.query (package:puro/src/env/engine.dart:83)
    <asynchronous suspension>
    #1      downloadSharedEngine (package:puro/src/env/engine.dart:182)
    <asynchronous suspension>
    #2      setUpFlutterTool.<anonymous closure> (package:puro/src/env/flutter_tool.dart:56)
    <asynchronous suspension>
    #3      checkAtomic.<anonymous closure> (package:puro/src/file_lock.dart:228)
    <asynchronous suspension>
    #4      lockFile (package:puro/src/file_lock.dart:62)
    <asynchronous suspension>
    #5      checkAtomic (package:puro/src/file_lock.dart:226)
    <asynchronous suspension>
    #6      setUpFlutterTool (package:puro/src/env/flutter_tool.dart:50)
    <asynchronous suspension>
    #7      createEnvironment (package:puro/src/env/create.dart:172)
    <asynchronous suspension>
    #8      CommandRunner.runCommand (package:args/command_runner.dart:212)
    <asynchronous suspension>
    #9      main (package:puro/src/cli.dart:224)
    <asynchronous suspension>
```

## `puro ls`, `puro use`, etc
```zsh
[E] Exception while querying Flutter version
[E] Assertion failed: "Unexpected result from sysctl: `1\n`"
    #0      EngineBuildTarget.query (package:puro/src/env/engine.dart:83)
    <asynchronous suspension>
    #1      downloadSharedEngine (package:puro/src/env/engine.dart:182)
    <asynchronous suspension>
    #2      setUpFlutterTool.<anonymous closure> (package:puro/src/env/flutter_tool.dart:56)
    <asynchronous suspension>
    #3      checkAtomic.<anonymous closure> (package:puro/src/file_lock.dart:228)
    <asynchronous suspension>
    #4      lockFile (package:puro/src/file_lock.dart:62)
    <asynchronous suspension>
    #5      checkAtomic (package:puro/src/file_lock.dart:226)
    <asynchronous suspension>
    #6      setUpFlutterTool (package:puro/src/env/flutter_tool.dart:50)
    <asynchronous suspension>
    #7      runFlutterCommand (package:puro/src/env/command.dart:28)
    <asynchronous suspension>
    #8      runOptional (package:puro/src/logger.dart:139)
    <asynchronous suspension>
    #9      getEnvironmentFlutterVersion (package:puro/src/env/version.dart:246)
    <asynchronous suspension>
    #10     listEnvironments (package:puro/src/env/list.dart:122)
    <asynchronous suspension>
    #11     CommandRunner.runCommand (package:args/command_runner.dart:212)
    <asynchronous suspension>
    #12     main (package:puro/src/cli.dart:224)
    <asynchronous suspension>

[E] Exception while querying Flutter version
[x] Assertion failed: "Unexpected result from sysctl: `1\n`"
    #0      EngineBuildTarget.query (package:puro/src/env/engine.dart:83)
    <asynchronous suspension>
    #1      downloadSharedEngine (package:puro/src/env/engine.dart:182)
    <asynchronous suspension>
    #2      setUpFlutterTool.<anonymous closure> (package:puro/src/env/flutter_tool.dart:56)
    <asynchronous suspension>
    #3      checkAtomic.<anonymous closure> (package:puro/src/file_lock.dart:228)
    <asynchronous suspension>
    #4      lockFile (package:puro/src/file_lock.dart:62)
    <asynchronous suspension>
    #5      checkAtomic (package:puro/src/file_lock.dart:226)
    <asynchronous suspension>
    #6      setUpFlutterTool (package:puro/src/env/flutter_tool.dart:50)
    <asynchronous suspension>
    #7      upgradeEnvironment (package:puro/src/env/upgrade.dart:148)
    <asynchronous suspension>
    #8      CommandRunner.runCommand (package:args/command_runner.dart:212)
    <asynchronous suspension>
    #9      main (package:puro/src/cli.dart:224)
    <asynchronous suspension>
```

**Comment by PixelToast** [comment]
This should be fixed in 1.1.13, thanks for reporting!

**Comment by damywise** [comment]
I've confirmed that it works perfectly now. Thanks for the fix!
